.github/workflows/link_startrek.yml

```diff
@@ -24,3 +25,4 @@ jobs:
                "Link": "<${{ github.server_url }}/${{ github.repository }}/pull/${{ github.event.number }}>; rel=\"relates\"",
                "Authorization": "OAuth ${{ secrets.OAUTH_STARTREK }}"
             }
+          ignoreStatusCodes: 409
```

Comment:
Checked that it works as expected:
https://github.com/yandex/ch-backup/actions/runs/5623893915/job/15239550426?pr=14